src/test/java/com/wikia/webdriver/elements/communities/desktop/components/spamwikireviewsubpages/AddQuestionableWikiSubpage.java

```diff
@@ -0,0 +1,23 @@
+package com.wikia.webdriver.elements.communities.desktop.components.spamwikireviewsubpages;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AddQuestionableWikiSubpage extends BasePageObject {
+
+  @FindBy(css = "#ids")
+  private WebElement addIdsTextArea;
+
+  @FindBy(xpath = "//button[@type='submit']")
+  private WebElement submitIdsButton;
+
+  public WebElement getSubmitIdsButton() {
```

Comment:
It's better not to expose WebElements in general, so please rewrite both methods to something like isTextAreaVisiblethat will return bool based on is displayed property

Reply:
Replaced with isSubmitIdsButtonDisplayed :)